vm/devices/storage/nvme_spec/src/nvm.rs

```diff
@@ -15,7 +15,7 @@ use zerocopy::LE;
 use zerocopy::U16;
 
 #[repr(C)]
-#[derive(Debug, AsBytes, FromBytes, FromZeroes, Inspect)]
+#[derive(Debug, AsBytes, FromBytes, FromZeroes, Inspect, Clone)]
```

Comment:
It will be needed when Restore path is merged on top of this. Let me double check. I assume that you'd prefer to merge this line along with Restore path?

Reply:
Yes please.